style.css

```diff
@@ -0,0 +1,261 @@
+body {
+	margin: 0;
+	font-family:'Montserrat-Regular';
+	font-style: normal;
+	box-sizing: border-box;
+	background: #E5E5E5;
+	min-width: 360px;
+}
+.wrap {
+    background: linear-gradient(to right, black 496px, white 19%);
+    height: 51vh;
+ }
+.header {
+	display: flex;
+	align-items: center;
+	height: 100px;
+	margin-bottom: 36px;
+}
+.header__logo {
+	display: flex;
+    padding-top: 35px;
+    padding-left: 62px;
+    margin-right: 265px;
+    margin-bottom: 12px;
+	cursor: pointer;
+}
+.header__logo-img {
+	width: 94px;
+	height: 25px;
+}
+.header__lang {
+	margin-right: 1000px;
+	margin-bottom: -20px;
+	cursor: pointer;
+}
+.header__lang-ru {
+	width: 31px;
+	height: 25px;
+	font-family: 'Montserrat-Medium';
+	font-weight: 200;
+	font-size: 21px;
+	line-height: 24px;
+	color: white;
+}
+.lang__vector {
+	width: 13px;
+	height: 7.7px;
+}
+.header__menu {
+	position: relative;
+    display: flex; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=5633356)